src/llm/apis/openai_completions.hpp

```diff
@@ -172,6 +171,8 @@ class OpenAIChatCompletionsHandler {
     std::optional<std::string> getPrompt() const;
     std::optional<int> getNumReturnSequences() const;
     StreamOptions getStreamOptions() const;
+    std::string& getProcessedJson();
```

Comment:
does it really need to be non-const? if there is any reason for that, why do we need getters if we can edit it anyway?